src/main/java/com/ezel/voza/domain/member/entity/enums/KickOutTime.java

```diff
@@ -3,7 +3,9 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 
 public enum KickOutTime {
-    SEVEN_DAY_STOP, THIRTY_DAY_STOP, ALL_DAY_STOP;
+    FIVE_HOUR_STOP, EIGHT_HOUR_STOP, TWELVE_HOUR_STOP,
+    ONE_DAY_STOP, THREE_DAY_STOP, SEVEN_DAY_STOP, THIRTY_DAY_STOP,
+    ALL_DAY_STOP;
```

Comment:
```suggestion
    ONE_DAY_STOP, THREE_DAY_STOP, SEVEN_DAY_STOP, THIRTY_DAY_STOP, ALL_DAY_STOP
```

같은 DAY니까 같은 열에 묶거나 아니면 아예 다른 명으로 하는게 낮겠습니다

Reply:
수정했습니다.